configs/test/bot/setup/android.bash

```diff
@@ -98,7 +101,11 @@ rm -rf clusterfuzz
 $GSUTIL_PATH/gsutil cp gs://$DEPLOYMENT_BUCKET/linux.zip clusterfuzz-source.zip
 unzip -q clusterfuzz-source.zip
 
-echo "Running ClusterFuzz."
-OS_OVERRIDE="ANDROID" ANDROID_SERIAL="$ANDROID_SERIAL" PATH="$PATH" NFS_ROOT="$NFS_ROOT" GOOGLE_APPLICATION_CREDENTIALS="$GOOGLE_APPLICATION_CREDENTIALS" ROOT_DIR="$ROOT_DIR" PYTHONPATH="$PYTHONPATH" GSUTIL_PATH="$GSUTIL_PATH" python $ROOT_DIR/src/python/bot/startup/run.py &
-
-echo "Success!"
+if [ -z "$ANDROID_SERIAL" ]; then
+  echo "No \$ANDROID_SERIAL set. Will automatically detect devices and start ClusterFuzz for each."
+  for serial in `$ADB_PATH/adb devices | awk -F' ' '{ print $1 }' | egrep -v '^(|List)$'`; do
```

Comment:
$1 looks wrong, it is getting serial and not usb: version ?

Reply:
I don't understand the issue here. This was meant to get the true serial instead of usb:<id> to address the previous comment:

> since this is open-source version, i think it should rely on actual serial than the usb version.

Reply:
ah right, reread your comment, you mean to keep this.

Reply:
Right, I'll leave it as-is for now since this is probably better for the general case. We can check in a modified version in our config repo for our startup script if needed, and if they diverge too much we can get rid of this change later.